services/integration.sh

```diff
@@ -73,13 +73,14 @@ run galley ${yellow} Info
 run gundeck ${blue} Info
 run cannon ${orange} Info
 run cargohold ${purpleish} Info
+run spar ${white} Info
```

Comment:
This is gonna look interesting on my terminal :) I'd be very 🙇 if we could find sth other than white

Reply:
aee6d83cbb26021665129f959dee3e2ec1701c43